app/api/websocket.py

```
@@ -34,20 +103,107 @@ async def audio_receiver(websocket: WebSocket, audio_q: "asyncio.Queue[Optional[
+        collected = []
+        async for sentence in llm.stream_generate_sentences(query, contexts):
+            if sentence and sentence.strip():
+                collected.append(sentence.strip())
+                yield sentence.strip()
+
+        # outputs를 명시적으로 남기고 싶으면 이렇게
+        if run is not None:
+            try:
+                run.end(outputs={"answer": " ".join(collected)})
+            except Exception:
+                pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)